test/integration/targets/lightsail/library/lightsail_keypair.py

```diff
@@ -1,79 +0,0 @@
-#!/usr/bin/python
```

Comment:
If Lightsail Key Pairs aren't the same as EC2 Key Pairs it would be nice to get this added as an additional module.  At which point it would make sense to adjust these integration tests to test two creation modes:
1) create a lightsail instance without specifying a key
2) create a lightsail instance with a key

Reply:
Sure. I can work on a module to manage lightsail keypairs. Although, there are some other higher priority lightsail modules needed IMO. The current module only manages lightsail instances. Lightsail also has databases, networks, storage that will need separate modules. In fact, the current lightsail module should be renamed to `lightsail_instance`.